index.html

```diff
@@ -65,10 +65,29 @@
       <!-- Example row of columns -->
       <div class="row">
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          
+          <!-- list of popular poetry -->
+          <h2>Popular Poetry</h2>
+          <ul class="poetry-list" style="list-style: none; padding-left: 0px;">
+            <li style="padding-bottom: 20px;">
+              <a href="https://www.poetryfoundation.org/poems-and-poets/poems/detail/45521">I Wandered Lonely as a Cloud</a>
+              <p><img class ="img-responsive" src="http://t0.gstatic.com/images?q=tbn:ANd9GcRa6CA5UmvvNVVWN6YSCa_TuO2I66V-HWjGhBEwzRyA3sRbD-41" alt="I wandered lonely as a cloud cover photo" style="height: 100px;"></p>
```

Comment:
Photo layout looks good. Only suggestion would be that class="img-responsive" might be a too generic of name for these photos and it might be better to name it something like poet-image with an id of walt whitman for example. I was thinking this might be a good change since there will be many photos we will use that will be responsive for the profile pictures and possibly other photos within the site.